apps/etterlatte-trygdetid/src/main/kotlin/trygdetid/TrygdetidService.kt

```diff
@@ -743,7 +758,7 @@ class TrygdetidServiceImpl(
         trygdetid: Trygdetid,
         brukerTokenInfo: BrukerTokenInfo,
     ): Trygdetid? {
-        if (trygdetid.ident == UKJENT_AVDOED) {
+        if (trygdetid.ident == UKJENT_AVDOED || trygdetid.tidligereFamiliepleier) {
```

Comment:
Er denne sjekken grunnen til at `tidligereFamiliepleier` må settes i trygdetid og lagres i db? Kunne det vært aktuelt å sjekke at ident == søker og dermed sluppet dette? Kan være at det gir andre issues altså, bare lurer

Reply:
Det å sjekke ident opp mot soeker burde gå fint, så endrer til det for å slippe enda et felt i db